vaadin-text-field.html

```diff
@@ -69,6 +69,10 @@
       .label-placeholder[no-label] {
         display: none;
       }
+
+      [part=error-message]:empty {
```

Comment:
Considering the use case that the error message is provided declaratively in advance, this CSS rule does not work (as the message is not empty).